exercises/chapter5/src/Data/Picture.purs

```diff
@@ -11,6 +11,12 @@ data Point = Point
   , y :: Number
   }
 
+instance pointEq :: Eq Point where
```

Comment:
I know the use of type classes here is looking ahead a little, but I figured it would be acceptable since the exercise solutions would not require the reader to know about them. This is to enable easier comparison in the unit test assertions.